modules/system/console/OctoberInstall.php

```diff
@@ -130,6 +131,25 @@ protected function setupAdvancedValues()
         $this->writeToConfig('app', ['debug' => $debug]);
     }
 
+    protected function askToInstallPlugins() {
+        $drivers = (bool)$this->confirm('Do you wish to install October.Drivers plugin?', false);
```

Comment:
Also, there is no need to assign a separate variable for this, just put the call in the conditional directly: `if ($this->confirm())`